include/grpc/grpc_security.h

```diff
@@ -131,6 +131,34 @@ grpc_credentials *grpc_google_iam_credentials_create(
     const char *authorization_token, const char *authority_selector,
     void *reserved);
 
+/* Function to be called by the metadata credentials plugin implementation when
+   the metadata is ready. */
+void grpc_credentials_plugin_metadata_notify(
```

Comment:
Sg. I mostly care about the prefixes being the same so that it's obvious that these things belong together.